samples/system-test/files.test.js

```diff
@@ -137,13 +141,72 @@ it('should make a file public', async () => {
   );
 });
 
-it('should generate a signed URL for a file', async () => {
+it('should generate a v2 signed URL for a file', async () => {
   const output = await exec(
     `${cmd} generate-signed-url ${bucketName} ${copiedFileName}`
   );
   assert.match(output, new RegExp(`The signed url for ${copiedFileName} is `));
 });
 
+it('should generate a v4 signed URL and read a file', async () => {
+  const output = await exec(
+    `${cmd} generate-v4-read-signed-url ${bucketName} ${copiedFileName}`
+  );
+
+  const regExp = new RegExp(
+    'Generated GET signed URL:\n' +
+      '(.*)\n' +
```

Comment:
folks will inevitably change the `console.log` without updating this test, so I think it will be a bit fragile.

Perhaps just assert that some of the content was output from the sample.

Reply:
good call, fixed!